cmd/neofs-ir/main.go

```diff
@@ -106,6 +106,7 @@ func main() {
 
 	select {
 	case <-ctx.Done():
+		log.Info("application is shutting down...")
```

Comment:
Maybe add a cause? Somewhat generic log.

Reply:
ok, added

Reply:
You can cancel with cause then. Currently it's just `signal.NotifyContext()`, but it can be something different.